extension-manifest-v3/scripts/build.js

```diff
@@ -75,6 +75,10 @@ if (argv.staging) {
   execSync('npm run download-engines', { stdio: 'inherit' });
 }
 
+if (manifest.permissions.includes('declarativeNetRequest') && argv.watch) {
```

Comment:
Please add this after L180, this is where we do some modifications to the result manifest. Now it is just between unrelated commands